/**:
  ros__parameters:
    type: moveit_twist_controller/MoveitTwistController
    group_name: arm_tcp_group # arm_with_gripper_cam_group
    gripper_joint_name: gripper_servo_joint
    twist_cmd_topic: /manipulator_arm_control/twist_cmd

    # moveit params
    kinematics_solver: "kdl_kinematics_plugin/KDLKinematicsPlugin"
    kinematics_solver_search_resolution: 0.005
    kinematics_solver_timeout: 0.05